ansible/roles/cartodb.app.dependencies/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+ - name: Install CartoDB apt dependencies
+   apt: name={{ item }} state=present
+   with_items:
+     - python-all-dev
+     - imagemagick
+     - unp
+     - zip
+
+ - name: Install ruby gem dependencies
+   apt: name={{ item }} state=present
+   with_items:
+     - libreadline6-dev
+     - openssl
+
+ - name: Install ruby gems
+   gem: name={{ item }} state=present user_install=no
+   with_items:
+     - bundler
+     - compass
+
+ - name: Add RAILS_ENV variable to global environment
+   lineinfile: dest=/etc/environment line="RAILS_ENV={{ cartodb_environment }}"
+
+ # TODO: Replace with bundler module if we want to only use ansible 2.0.0+
+ - name: Install ruby bundle
+   shell: bash -lc "bundle install" chdir={{ cartodb_dir }}
```

Comment:
This is something that was in the other cartodb provisioning repo, just copied it from there. I'll see about cleaning this up.